Makefile

```diff
@@ -175,10 +175,6 @@ deps-js:
 
 deps-go:
 	go mod download
-ifeq ($(shell uname -s),Linux)
-	# Dependency required for Linux Fleet Desktop.
-	sudo apt-get install gcc libgtk-3-dev libayatana-appindicator3-dev -y
-endif
```

Comment:
This PR should just remove this and nothing else :)

Sorry for all the confusion.

Reply:
No problem! Update coming up.